docs/jupyter.rst

```diff
@@ -281,40 +276,81 @@ Then we have to set up a data server to output the data in "tiles".
         'v': range(1, length+1),
     })
 
-    # get the tileset info (bounds and such) of the dataset
-    tsinfo = hgpo.tileset_info(df, 'x', 'y')
+    import higlass
+    import higlass.tilesets as hgti
+    import higlass.client as hgcl
 
-    ts = hgti.Tileset(
-        tileset_info=lambda: tsinfo,
-        tiles=lambda tile_ids: hgpo.format_data(
-                    hgut.bundled_tiles_wrapper_2d(tile_ids,
-                        lambda z,x,y,width=1,height=1: hgpo.tiles(df, 'x', 'y',
-                            tsinfo, z, x, y, width, height))))
+    ts = hgti.dfpoints(df, x_col='x', y_col='y')
 
-    # start the server
-    server = hfse.start([ts])
+    (display, server, viewconf) = higlass.display([
```

Comment:
It is not Pythonic to use brackets in tuple function outputs.

Reply:
FYI, there's a handy new feature for quickly fixing minor typos: the insert button (or cmd+g)

Reply:
PS I fixed it :)